### PR TITLE
✨ feat: Add `signMessage` method

### PR DESCRIPTION
**Pull request by insulineru**

### CODE REVIEW

File: src/context/WalletProvider.tsx [hunk 1]
```diff
@@ -851,6 +851,24 @@ const WalletProvider = function WalletProvider({ children }: { children: ReactNo
...
+    const currentState = walletState[currentName]
+
+    if (isEvmWallet(currentState, fromChainId)) {
+      const signer = currentState.provider!.getSigner()
+      return signer?.signMessage(message)
+    } else {
+      throw new Error(`[Wallet] signMessage error: ${currentName} is not supported`)
+    }
+  }
+
```
Comment (nrjshka) [inline src/context/WalletProvider.tsx:871]: Don't forget to support `try{} catch` for such cases when user rejected tx

**Review by nrjshka — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viaprotocol/web3-wallets",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (nrjshka) [inline package.json:3]: Why?

**Review by iower — APPROVED** [review]

File: src/types.ts [hunk 3]
```diff
@@ -111,6 +111,9 @@ type TWallet = {
       fromChainId?: number
     }
   ) => Promise<string /* | false */> // todo: sendTx reject => false
+  signMessage: (message: string, options?: {
+    fromChainId?: number
+  }) => Promise<string>
```
Comment (iower) [inline src/types.ts:116]: How about flatter? Seems like there won't be any more options.
```suggestion
  signMessage: (message: string, forChainId?: number) => Promise<string>
```

Comment (insulineru) [reply src/types.ts:116]: same options as `sendTx`. we don't know how this will expand in the future

**Pull request merged**